cloudformation/media-atom-maker-dev.yml

```diff
@@ -0,0 +1,53 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: "Media atom maker development"
+Parameters:
+  DynamoDBTablePrefix:
+    Description: "DynamoDB Table prefix"
+    Type: "String"
+    Default: "media-atom-maker"
+Resources:
+  MediaAtomGroup:
+    Type: "AWS::IAM::Group"
+    Properties:
+      Policies:
+        - PolicyName:   "Group-Policy"
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: "Allow"
+                Action: ["dynamodb:*"]
+                Resource:
+                  - Fn::Join:
+                    - ""
+                    -
+                      - "arn:aws:dynamodb:"
+                      - Ref: "AWS::Region"
+                      - ":"
+                      - Ref: "AWS::AccountId"
+                      - ":table/"
+                      - "Ref" : "MediaAtomsDynamoTable"
+  MediaAtomUser:
+    Type: "AWS::IAM::User"
+    Properties:
+      Path: "/"
+      Groups: [{ "Ref": "MediaAtomGroup" }]
+  HostKeys:
+    Type: "AWS::IAM::AccessKey"
+    Properties:
+      UserName: {"Ref": "MediaAtomUser"}
+  MediaAtomsDynamoTable:
+    Type: "AWS::DynamoDB::Table"
+    Properties:
+      AttributeDefinitions:
+        - AttributeName: "id"
+          AttributeType: "S"
+      KeySchema:
+        - AttributeName: "id"
+          KeyType: "HASH"
+      ProvisionedThroughput:
+        ReadCapacityUnits: "10"
+        WriteCapacityUnits: "5"
+Outputs:
+  EditsDynamoTable:
```

Comment:
I think you'd need another block for the secret as its behind a `Fn::GetAtt` http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-iam-accesskey.html ?